test/e2e/airgap.go

```diff
@@ -192,3 +194,30 @@ func runDockerAirgapConfigFlow(test *framework.ClusterE2ETest) {
 	test.DeleteCluster(framework.WithBundlesOverride(bundleReleasePathFromArtifacts))
 	test.ChangeInstanceSecurityGroup(os.Getenv(framework.RegistryMirrorDefaultSecurityGroup))
 }
+
+func runDockerAirgapUpgradeFromReleaseFlow(test *framework.ClusterE2ETest, latestRelease *releasev1.EksARelease, wantVersion anywherev1.KubernetesVersion, clusterOpts ...framework.ClusterE2ETestOpt) {
```

Comment:
is there anything specific to docker in this method?

Reply:
It changes the security group of the EC2 instances it runs in to "airgap" it, these are only run for the docker airgapped tests.

Reply:
ahh i see
does this mean you couldn't execute this method in another test using a different provider?

Reply:
You probably can assuming the other providers use an EC2 instance as an admin machine. Though, unlike the other providers the Docker clusters are created on the admin machine, so this makes sense only for this provider